keel-api/src/main/kotlin/com/netflix/spinnaker/config/YamlConfiguration.kt

```diff
@@ -15,12 +15,10 @@
  */
 package com.netflix.spinnaker.config
 
-import com.netflix.spinnaker.keel.yaml.YamlHttpMessageConverter
-import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
 
 @Configuration
 class YamlConfiguration {
```

Comment:
This doesn't do anything now.. any reason to keep it around?

Reply:
Good spot. Fixed